bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XToYH/NMSSM_XToYH_MX_500_MY_300_proc_card.dat

```diff
@@ -0,0 +1,22 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
```

Comment:
Hi, 
could you please remove those lines, some of them are duplicate.

Reply:
Hi, the lines are now removed